test/framework/fixtures/libgoalFixture.go

```diff
@@ -324,6 +328,22 @@ func (f *LibGoalFixture) ShutdownImpl(preserveData bool) {
 	}
 }
 
+// dumpLogs prints out node.log files for the running nodes
+func (f *LibGoalFixture) dumpLogs(dataDir string) {
+	file, err := os.Open(dataDir + "/node.log")
```

Comment:
please limit to ~100 lines, and dump algod-err.log and algod-out.log if not empty as well